agrona/src/main/java/org/agrona/ExpandableDirectByteBuffer.java

```diff
@@ -140,6 +140,17 @@ public void setMemory(final int index, final int length, final byte value)
         }
     }
 
+    public void copyMemory(final int index, final long fromAddress,
```

Comment:
A more in-keeping API would be to add both `putMemory` and `getMemory` methods to the respective interfaces and provide implementations.